### ISSUE TITLE
DROF and DATM stream names

### ISSUE THREAD

**Issue by aekiss** [COSIMA/om3-scripts#27]
[generate_xml_datm.py](https://github.com/COSIMA/om3-scripts/blob/main/data_stream_xml_generation/generate_xml_datm.py) produces stream names like this
```xml
  <stream_info name="CORE_IAF_JRA55do.PRSN">
```
which is confusing, because CORE is a different dataset from JRA55do, and also the same stream names are also used for the RYF configs. Can we drop `CORE_IAF_` and just have names like this?
```xml
  <stream_info name="JRA55do.PRSN">
```

Similarly, [generate_xml_drof.py](https://github.com/COSIMA/om3-scripts/blob/main/data_stream_xml_generation/generate_xml_drof.py) uses 
```xml
  <stream_info name="rof.iaf_jra">
```
but 
```xml
  <stream_info name="JRA55do.FRIVER">
```
(and similar for LICALVF - see https://github.com/COSIMA/om3-scripts/issues/29) would be less confusing and more consistent.

**Comment by dougiesquire** [comment]
I think we can change these here, but `CORE_IAF_JRA55do` is actually the name of the new datamode that @ezhilsabareesh8 set up, so if we don't like that name we should change it [here](https://github.com/COSIMA/access-om3/blob/c9214a250d065b13be8f942780ac0bf524fa6e77/CDEPS/patches/atm_comp_nuopc.F90.patch) also

**Comment by dougiesquire** [comment]
Closed by #31
